src/js/angular/clustermanagement/services/cluster-svg.service.js

```diff
@@ -0,0 +1,100 @@
+export class LineConfig {
+    constructor(point1, point2, strokeColor, strokeWidth, bgColor, dashArray) {
+        this.point1 = point1 || {x: 0, y: 0};
+        this.point2 = point2 || {x: 100, y: 0};
+        this.strokeColor = strokeColor || "var(--tertiary-color)";
+        this.strokeWidth = strokeWidth || 5;
+        this.bgColor = bgColor || "#EEEEEE";
+        this.dashArray = dashArray || "10, 10";
+    }
+
+    setPoint1(point1) {
+        this.point1 = point1;
+        return this;
+    }
+
+    setPoint2(point2) {
+        this.point2 = point2;
+        return this;
+    }
+
+    setStrokeColor(strokeColor) {
+        this.strokeColor = strokeColor;
+        return this;
+    }
+
+    setStrokeWidth(strokeWidth) {
+        this.strokeWidth = strokeWidth;
+        return this;
+    }
+
+    setBgColor(bgColor) {
+        this.bgColor = bgColor;
+        return this;
+    }
+
+    setDashArray(dashArray) {
+        this.dashArray = dashArray;
+        return this;
+    }
+}
+
+export class HexagonConfig {
+    constructor(classes = '', size = 50, color = '#EEEEEE') {
+        this.classes = classes;
+        this.size = size;
+        this.color = color;
+    }
+
+    setClasses(classes) {
+        this.classes = classes;
+        return this;
+    }
+
+    setSize(size) {
+        this.size = size;
+        return this;
+    }
+
+    setColor(color) {
+        this.color = color;
+        return this;
+    }
+}
+
+export class ClusterSvgService {
+    static getLeaderSvg(config = new HexagonConfig()) {
+        return `<svg width="${config.size}" height="${config.size}" viewBox="-25 -25 50 50">
```

Comment:
And now we have two dofferent ways to draw things in the cluster view. How is this better than using just d3.js?